### PR TITLE
Create todo structure

### PR DESCRIPTION
**Pull request by denscholar**
## Created the todo structure based on the following requirement:
- Set up a new project with webpack that is based on the webpack exercise you have already completed.

- Create an index.html file and write your HTML markup here. Create an empty To-Do List placeholder div or ul element. The index.html file must be set as a template using the HTML Webpack Plugin.

- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].

- Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

- Create a style.css and set rules for the To-Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,21 +11,25 @@
...
+        <input type="text" name="todo-input" id="todo-input" placeholder="Type in your task here">
+        <button type="submit" id="task-submit"><i class="fas fa-level-down-alt rotate"></i></button>
+    </form>
+    <ul class="task-list">
+      <!-- <li class="task">
+        <input class="check" type="checkbox" value="">
+        <input type="text" class="task-text" type="checkbox" value="This is the first task" readonly>
+        <button><i id="" class="fas fa-ellipsis-v"></i></button>
+        <button class="btn"><i id="taskdelete" class="fas fa-trash"></i></button>
+      </li> -->
```
Comment (hiromataba) [inline src/index.html]: - [ ] Please remove commented code, it makes your code look messy so kindly remove it to make your code clean

File: src/style.css [hunk 2]
```diff
@@ -1,100 +1,102 @@
...
-  font-size: 22px;
+  font-size: 20px;
   border: none;
   border-top: 1px solid rgb(197, 196, 196);
   border-bottom: 1px solid rgb(187, 187, 187);
   background: rgb(247, 245, 245);
   outline: none;
+}
 
-  /* width: 100%; */
```
Comment (hiromataba) [inline src/style.css:55]: - [ ] Please remove commented code, it makes your code look messy so kindly remove it to make your code clean

File: dist/index.html [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
+        <input type="text" name="todo-input" id="todo-input" placeholder="Type in your task here">
+        <button type="submit" id="task-submit"><i class="fas fa-level-down-alt rotate"></i></button>
+    </form>
+    <ul class="task-list">
+      <!-- <li class="task">
+        <input class="check" type="checkbox" value="">
+        <input type="text" class="task-text" type="checkbox" value="This is the first task" readonly>
+        <button><i id="" class="fas fa-ellipsis-v"></i></button>
+        <button class="btn"><i id="taskdelete" class="fas fa-trash"></i></button>
+      </li> -->
```
Comment (hiromataba) [inline dist/index.html]: - [ ] Please remove commented code, it makes your code look messy so kindly remove it to make your code clean

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @denscholar :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_sports: 
- No linter errors :+1: 
- Nice PR description and title :heavy_check_mark: 
- Nice webpack setup :heavy_check_mark: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] As you are using webpack please  add commands in your README file to install npm `npm install` and to start your project in the browser `npm start`
  
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 3 above)
Comment (denscholar) [reply dist/index.html]: Done, Thank you.

**Review by hiromataba — APPROVED** [review]
## STATUS : APPROVED :tada: :tada: :1st_place_medal: 
Hi :wave:  @denscholar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**